tasks/dataset-split/src/split_image_dataset.py

```diff
@@ -10,38 +10,45 @@ def splitImageDataset(originalDataset: ImageDataset, datasetParts: int, taskRunI
     splitDatasets: list[ImageDataset] = []
 
     for index, sampleChunk in enumerate(splitSamples):
-        splitDataset = ImageDataset.createDataset(f"{taskRunId}-split-dataset-{index}", projectId)
```

Comment:
remove taskRunId from function parameters since its not used anywhere in the function